Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -1,10 +1,11 @@
 import Foundation
 import UIKit
 import BraintreePayPal
+import BraintreeCore
 
 class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
 
-    lazy var payPalClient = BTPayPalClient(apiClient: apiClient)
+    lazy var payPalClient = BTPayPalClient(apiClient: BTAPIClient(authorization: "sandbox_jy4fvpfg_v7x2rb226dx4pr7b")!)
```

Comment:
Oh, actually we only want this in the method: `universalLinkFlow` not for all of the PayPal flows. Sorry didn't notice that until after I approved.

Reply:
oops! I've updated it now